blockchain/state_transition.go

```diff
@@ -368,65 +368,11 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 	// - reset transient storage(eip 1153)
 	st.state.Prepare(rules, msg.ValidatedSender(), msg.ValidatedFeePayer(), st.evm.Context.Coinbase, msg.To(), vm.ActivePrecompiles(rules), msg.AccessList())
 
-	// SetCode sender nonce increment should be done before set code process.
-	if msg.Type() == types.TxTypeEthereumSetCode {
-		st.state.IncNonce(msg.ValidatedSender())
-	}
-
-	// Check authorizations list validity.
+	// Check authorization list validity.
```

Comment:
I agree that it's not just about checking validity.

> At the start of executing the transaction, after incrementing the sender's nonce, for each [chain_id, address, nonce, y_parity, r, s] tuple do the following:

Are you quoting from this sentence?
I'm afraid it's about the timing of processing the set code.
Since the function name will be changed to `processAuthorizationList`, how about clarifying `set code`?

```
// Check authorization list validity and set code.
```

Reply:
It's fixed by https://github.com/kaiachain/kaia/pull/145/commits/5e9339815336f02bf2a66048b86eaac3727e28a2.